test_voc.ino

```diff
@@ -0,0 +1,17 @@
+#if false
+
+void setup() {
+  Serial.begin(9600);
+  while (!Serial) { /* Wait for Serial monitor to connect */ }
+  Serial.println("Connecting to sensor...");
```

Comment:
I don't think this line makes sense for this one? Since there's no `initialize` function to call

Reply:
agreed

Reply:
What line do you mean?

Reply:
Line 6 (`Serial.println("Connecting to sensor...");`) -- the line I left the comment on

Reply:
Oh I didn't see that the comment was on line 6 specifically